ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpTransactionSignerTest.java

```diff
@@ -12,33 +12,33 @@
  */
 package tech.pegasys.ethsigner.signer.hashicorp;
 
-import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
 
 import tech.pegasys.ethsigner.TransactionSignerInitializationException;
 
-import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 public class HashicorpTransactionSignerTest {
 
-  private final HashicorpSignerFactory hashicorpSignerFactory = new HashicorpSignerFactory();
+  private final HashicorpSigner hashicorpSigner = new HashicorpSigner();
 
   @Test
-  public void vaultTimingOut() throws IOException {
-
-    final File authFile = createFile();
-
+  public void vaultTimingOut(@TempDir final Path tempDirectory) throws IOException {
     assertThatThrownBy(
             () ->
-                hashicorpSignerFactory.createSigner(
-                    new HashicorpConfig(
-                        "signingKeyPath", "serverHost", 877, authFile.toPath(), 1L)))
+                hashicorpSigner.createSigner(
+                    new HashicorpConfig.HashicorpConfigBuilder()
```

Comment:
nit: didn't _need_ to change to use the builder

Reply:
🤷‍♂ allows me to selectively set the params.